src/components/radios/template.njk

```diff
@@ -47,6 +47,9 @@
     {% for item in params.items %}
     {% set id = item.id if item.id else idPrefix + "-" + loop.index %}
     {% set conditionalId = "conditional-" + id %}
+    {%- if item.divider %}
+    <span class="govuk-radios__divider">{{ item.divider }}</span>
```

Comment:
Should this just be a `div`, instead of `span` that has `display: block` set. It would make it consistent with the markup of the radios.

Reply:
it can be a div